test/node/test_farm.rb

```diff
@@ -26,6 +26,15 @@
 require_relative '../../lib/zold/node/farm'
 
 class FarmTest < Minitest::Test
+  class TestLogger
```

Comment:
@emilianodellacasa it's not clear from the name that it saves the last error message. So, when you reference it later, I have to guess, wtf is going on.